provisions/roles/cleanup/jenkins/slave/tasks/main.yml

```diff
@@ -1,21 +1,28 @@
 ---
```

Comment:
Using `with_items`, we can make this a single task. Is there a reason we're doing this as separate jobs for each container?

Reply:
that was extension of the previous cleanup code. fixing this